<launch>
        <node pkg="tf" type="static_transform_publisher" name="world2odom"
              args="0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 1.0000 world orb_slam 100"/>
    <!--    <node pkg="tf" type="static_transform_publisher" name="orbodom2odom"-->
    <!--          args="0.0000 0.0000 0.0000 0.0000 0.0000 0.0000 1.0000 orb_odom odom 100"/>-->
    <param name="/ORBSLAM3_tightly/out_path" value="/home/da/project/ros/orb_dvl2_ws/src/dvl2/orb3_result/"/>
    <param name="/ORBSLAM3_tightly/map_file" value="/home/da/project/ros/orb_dvl2_ws/src/dvl2/orb3_result/Altlas.osa"/>
    <param name="/ORBSLAM3_tightly/is_load_map" value="true" type="bool"/>
    <node pkg="ORB_DVL2" type="stereo_dvl2" name="stereo_dvl"
          args="$(find ORB_DVL2)/Vocabulary/ORBvoc.txt $(find ORB_DVL2)/data/underwater_orbslam3_falcon.yaml false" output="screen"/>

<!--    <include  file="$(find pose_recorder)/launch/redord.launch"/>-->
<!--    <node pkg="ORB_SLAM3_DVL_tightly" type="cameras_black_node.py" name="camera_black">-->
<!--        <remap from="/img_l/compressed" to="/CAM_L/image_dehazed/compressed"/>-->
<!--        <remap from="/img_r/compressed" to="/CAM_R/image_dehazed/compressed"/>-->
<!--    </node>-->

    <!--    <node pkg="sonar_localization" type="sonar_black_node.py" name="ekf_se" clear_params="true" output="screen"/>-->
        <arg name="model" default="$(find ORB_DVL2)/urdf/bluerov.urdf"/>
        <param name="robot_description" command="$(find xacro)/xacro $(arg model)" />
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find ORB_DVL2)/launch/falcon_tightly.rviz"/>

</launch>